src/Service/SF1500/SF1500.php

```diff
@@ -420,58 +396,135 @@ public function getEnhedNavnOgOverordnetOrganisationsId($organisationEnhedsId, $
     }
 
     /**
-     * Fetches bruger and organisation funktions id for nearest manager or null if no manager exists.
+     * Fetches bruger and organisation funktions id for managers from user id.
+     * Returns null if no manager exists.
```

Comment:
Add a return type the make this explicit, i.e.

```php
…): ?array
```

Reply:
Modified to always return array. Empty if none found.